daft/expressions/expressions.py

```diff
@@ -1192,6 +1377,7 @@ def extract(self, pattern: str | Expression, index: int = 0) -> Expression:
             ├╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌┤
             │ 345-678 ┆ 345     │
             ╰─────────┴─────────╯
+            #FLAGGED
```

Comment:
What's the error here?

Reply:
I think it's with the regex pattern, I'll try to make it simpler and easier to read